daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
```

Comment:
@ibuildthecloud is absolute right.
To load the kernel modules `bridge` and `nf_nat` you have to invoke `modprobe bridge` and `modprobe nf_nat` as two seperate commands. With this implementation here only `bridge` will be loaded and the string "nf_nat" will be sent as a module option while loading the `bridge` module. 
For more details about using `modprobe` I strongly recommend to read the [man pages](http://linux.die.net/man/8/modprobe).